apps/extension/src/core/media/openrouter.ts

```diff
@@ -0,0 +1,46 @@
+import { getExternalConfigURL } from "~core/utils/utils"
+import type { MediaModelConfig, RequestOptions } from "./model"
+import { MediaModel } from "./model"
+
+
+export function init(
+  config: Pick<MediaModelConfig, "debug" | "identifier"> &
+    Partial<Pick<MediaModelConfig, "cacheGet" | "cacheSet">>,
+  opts: RequestOptions
+): MediaModel {
+  return new MediaModel(
+    {
+      ...config,
+      defaultBaseUrl: `${getExternalConfigURL()}/v1/api`,
+      getPath: () => "/media/generations",
+      transformForRequest: (req, meta) => {
+        const {
+          mimeType,
```

Comment:
`mimeType` should be snake-cased here for consistency

Reply:
if easier, we can remove it, too, since customizing it isn't supported right now

Reply:
agreed, changing!